install_files/ansible-base/roles/install-fpf-repo/tasks/upgrade_apt.yml

```diff
@@ -0,0 +1,33 @@
+---
+# Fix for CVE-2019-3462 (see https://justi.cz/security/2019/01/22/apt-rce.html)
+# If apt < 1.4.9, it is vulnerable to CVE-2019-3462 and we must ensure no
+# redirects are followed when updating apt via apt.
+- name: Upgrade apt without following redirects
+  shell : |
+    apt -o Acquire::http::AllowRedirect=false update
```

Comment:
Is there a reason we don't "fail fast" by using `set -e` or `&&` here so that if the update fails, we don't even attempt the latter steps?

Reply:
good point, added `&&`